sem_server_ros/Triple old_triple
---
std_msgs/String feedback
